charts/schema-loading/templates/batchjob-schema-loading.yaml

```diff
@@ -76,5 +89,20 @@ spec:
               key: db-password
         - name: SCHEMA_TYPE
           value: "{{ .Values.schemaLoading.schemaType }}"
+        {{- if .Values.schemaLoading.secretName }}
+        envFrom:
+        - secretRef:
+            name: "{{ .Values.schemaLoading.secretName }}"
+        {{- end }}
```

Comment:
If you set arbitrary sensitive data like credentials as a `Secret` resource, those values are passed to the container as environment variables.